### DISCUSSION TITLE
Question about relative spatial coordinate system

### DISCUSSION THREAD

**Discussion by mengyuan404** [PaddlePaddle/PaddleSpatial#179] [BaiduKDDCup2022] [answered]
Could you provide the relation between the spatial coordinate system and standard geographic direction，such as the angle between y-axis and North?

**Comment by ethanyxx** [comment, marked as answer]
Hi,

For the sake of privacy concerns, we can not provide the transformation matrix between our spatial reference and any known geographic system. The spatial transformation we adopted does not change the direction or relative position of wind turbines.
